protocol-definitions/CPMap.yaml

```diff
@@ -236,3 +236,44 @@ methods:
           since: 2.7
           doc: |
             True if key was associated with newValue, otherwise false.
+  - id: 7
+    name: putIfAbsent
+    since: 2.7
+    doc: |
+      Puts the key-value into the specified map if the key is not currently associated with a value.
+    request:
+      retryable: false
+      partitionIdentifier: -1
+      params:
+        - name: groupId
+          type: RaftGroupId
+          nullable: false
+          since: 2.7
+          doc: |
+            CP group ID of this CPMap instance.
+        - name: name
+          type: String
+          nullable: false
+          since: 2.7
+          doc: |
+            Name of this CPMap instance.
+        - name: key
+          type: Data
+          nullable: false
+          since: 2.7
+          doc: |
+            Key of the value.
+        - name: value
+          type: Data
+          nullable: false
+          since: 2.7
+          doc: |
+            Value to associate with the key.
+    response:
+      params:
+        - name: response
+          type: Data
+          nullable: true
+          since: 2.7
+          doc: |
+            Value associated with the key if already present, otherwise null.
```

Comment:
this is old value right? you may want to clarify it even more by adding old at the beginning